entity-framework/core/miscellaneous/events/interception.md

```diff
@@ -0,0 +1,750 @@
+---
+title: Interceptors - EF Core
+description: Interception for database operations and other events
+author: ajcvickers
+ms.date: 10/08/2020
+uid: core/miscellaneous/events/interception
+---
+
+# Interceptors
+
+Entity Framework Core (EF Core) interceptors enable interception, modification, and/or suppression of EF Core events. This includes low-level database operations such as executing a command, as well as higher-level events, such as calls to SaveChanges.
```

Comment:
```suggestion
Entity Framework Core (EF Core) interceptors enable interception, modification, and/or suppression of EF Core operations. This includes low-level database operations such as executing a command, as well as higher-level ones such as calls to SaveChanges.
```

(because events mean .NET events which are also a thing)